src/index.ts

```diff
@@ -16,10 +22,21 @@ export interface Services {
 export const buildServicesFromConfig = async (): Promise<Services> => {
     let ceramic
     if (config.ceramic.mode == "client") {
+        console.log("Creating ceramic via http client")
         ceramic = new CeramicClient(config.ceramic.apiURL, { docSyncEnabled: true, docSyncInterval: 500 })
     } else if (config.ceramic.mode == "node") {
-        // TODO
-        throw new Error ("not supported yet")
+        console.log("Creating ceramic via local node")
+        basicsImport.multicodec.add(dagJose)
+        const format = legacy(basicsImport, dagJose.name)
+
+        const ipfs: IpfsApi = ipfsClient({url: config.ceramic.ipfsApi, ipld: { formats: [format] } })
```

Comment:
Why do you run an ipfs client rather than an in-process ipfs? How do you spin this client up? Doesn't seems like it's part of the setup scripts you've created.

Reply:
that's right, these tests rely on any external services (ipfs, ceramic, cas, etc) to be already up and running when the tests start.  We don't yet have anything written for actually coordinating that process, that's the docker instances side that val and zach are working on

Reply:
We can add support for running an in-process ipfs, but it was just easier to start with the ipfs being external